lib/snowplow_tracker/emitters/cache.ex

```diff
@@ -0,0 +1,76 @@
+defmodule SnowplowTracker.Emitters.Cache do
+  @moduledoc """
+  Module which implements functions to interact with ETS
+  """
+
+  require Logger
+
+  alias :ets, as: Ets
+
+  @table Application.get_env(:snowplow_tracker, :table)
+  #@persist_every Application.get_env(:snowplow_tracker, :persist_every, 5000)
+  @path Application.get_env(:snowplow_tracker, :path, ".")
+  @lock "lock"
+
+  def init(table \\ @table) do
+    PersistentEts.new(
+      table,
+      "#{@path}/#{Atom.to_string(table)}.tab",
+      [:public, :set, :named_table, {:read_concurrency, true}]
+    )
+
+    Logger.log(:debug, "Table #{table} created successfully!")
+    {:ok, :success}
+  end
+
+  def insert(payload, table \\ @table) do
+    Ets.insert(table, payload)
+    {:ok, :success}
```

Comment:
are you pattern matching on `{:ok, :success}` anywhere else? If not just return this - 

```elixir
{
  :ok, 
  Ets.insert(table, payload)
}
```
which is consistent to other functions below.